changelog/_unreleased/2023-09-20-avoid-creating-seo-urls-for-headless-sales-channels.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Avoid creating SEO URLs for headless sales channels.
+issue: NEXT-00000
+flag: v6.6.0.0
```

Comment:
not sure if this should be v6.6.0.0 or V6_6_0_0 because the env variable needs the second format but the flag itself is with dots.